### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by AbdulSalamAbro**
* Added a clickable Popup menu in the mobile version of the Portfolio website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,19 +14,20 @@
...
+        <a href="#" id="closebtn">&times;</a>
         <nav id="desktop-navbar">
-            <a href="#home">Portfolio</a>
-            <a href="#About">About</a>
-            <a href="#Contact">Contact</a> 
-    </nav>
-    </header>
+            <a href="#portolioworks" id="menu-link1">Portfolio</a>
+            <a href="#About" id="menu-link2">About</a>
+            <a href="#Contact" id="menu-link3">Contact</a>
```
Comment (BregornOriginal) [inline index.html]: - [ ] Kindly modify the `href=` to navigate to the correct part of the portfolio when an option in the mobile menu it's clicked

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team 

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

hand-sketching-stair-slogan-one-step-closer-chalkboard-79948193

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AbdulSalamAbro) [reply index.html]: sure I am just updating it

**Review by shahadat3669 — APPROVED** [review]
Hi @AbdulSalamAbro, @Al-Amin-Khan-Shakil, 

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No linter Errors
✅ Descriptive PR Title
✅ Descriptive PR Description


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- [Optional] 👍 Good job on implementing the navigation items and making the links functional! I noticed that the mobile menu holder div has a white background and covers the full width, but it's missing the full height as per the design. Please follow the design carefully and update your code to match the design. 👍 

| Your | Design |
| --- | --- |
| image | image |


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**